### DISCUSSION TITLE
Help in setup

### DISCUSSION THREAD

**Discussion by namratast** [getsops/sops#1573] [General]
Hello, 
is there anyone who can help me with download and install sops?
I'm just not able to get along.
Thank you!

**Comment by devstein** [comment]
👋 hi @namratast can you share more about what you tried and what isn't working?

**Reply by namratast** [reply]
Hi @devstein I downloaded a binary for windows and followed the instructions provided in the link below:
https://fluxcd.io/flux/guides/mozilla-sops/
Encrypting secrets using age

**Reply by devstein** [reply]
What error did you run into?
